ui/datalist.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const DataList = ({data}) => {
+    return (
+        <datalist id="data-from-firestore">
+            {data.map((item, i) => (
+                <option key={i} value={item.value}>{item.label}</option>
+            ))}
+        </datalist>
+    )
+}
+export default DataList;
```

Comment:
Remove `default` export. We want to embrace export barrels, and default exports make this more difficult to maintain.

```suggestion
```